icepll/icepll.cc

```diff
@@ -25,6 +25,20 @@
 #include <emscripten.h>
 #endif
 
+enum feedback_mode
+{
+	FEEDBACK_SIMPLE,
+	FEEDBACK_NON_SIMPLE,
+	FEEDBACK_PHASE_AND_DELAY,
+};
+
+struct freq_constraint {
```

Comment:
consider using a `std::pair<double, double>` rather than defining a custom struct for this,